debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleDdlParser.java

```diff
@@ -298,6 +309,19 @@ private int getVarCharDefaultLength() {
             return 4000;
         }
 
+        private void setPrecision(Precision_partContext precisionPart, ColumnEditor columnEditor) {
```

Comment:
This code (https://github.com/gunnarmorling/debezium-incubator/blob/078a18337cb2bb0fb91faca2b06a8b8869342080/debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleDdlParser.java#L250) is porbably made obsolete by these two methods

Reply:
Indeed; I had meant to remove it. Will add it.

Reply:
Indeed; I had meant to remove it. Will do it.

Reply:
I am sorry but my proposal managed to break all other datatypes, see
```
Failed tests: 
  OracleDdlParserTest.shouldParseCreateTable:57 expected:<[1]000> but was:<[4]000>
```

Reply:
Argh, ok. Looking. Sorry for not testing properly locally :( Which reminds me that we really need a PR builder for the incubator repo, including the Oracle connector.

Reply:
Ok, reworked that now, the precision is also applied to `(N)VARCHAR2` columns now. The build is passing with that.

Reply:
Thanks, just a one nitpick - I think `TIMESTAMP` has the same issue?

Reply:
Added a commit for that, too. Any other types where you see it missing?